plugins/outputs/file/file_test.go

```diff
@@ -185,17 +175,18 @@ func TestFileStdout(t *testing.T) {
 func createFile(t *testing.T) *os.File {
 	f, err := os.CreateTemp("", "")
 	require.NoError(t, err)
+	t.Cleanup(func() {
+		require.NoError(t, f.Close())
+		require.NoError(t, os.Remove(f.Name()))
```

Comment:
Why not create these in t.TempDir() and let the testing package delete them?

Reply:
Amended in 7279328d3234e91eecc96daf2047eca1d058f4b6